plugins/Login/lang/en.json

```
@@ -83,6 +86,16 @@
         "CancelPasswordResetRequestRemoved": "The request has been removed",
         "CancelPasswordResetRequestRemovedMessage": "We have recorded that you did not request this password reset. The reset request has been removed, and no changes have been made to your account.",
         "CancelPasswordResetSecurityTip": "For added protection, we recommend enabling two-factor authentication (2FA).",
-        "SecurityTip": "Security Tip"
+        "SecurityTip": "Security Tip",
+        "LoginFromDifferentCountryEmailSubject": "Unusual login activity detected on your Matomo account",
+        "LoginFromDifferentCountryEmailBody01": "We’ve detected a login to your Matomo account from a location that is different from your usual login area.",
```

```suggestion
        "LoginFromDifferentCountryEmailBody01": "We’ve detected a login to your Matomo account from a location that is different from your previous login area.",
```
We shouldn't write usual if we don't know it. For that we would need to store where and how often someone logs in from.